src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -460,7 +496,61 @@ llvm_verify_x86 (Some (llvmModule :: LLVMModule x)) path nm globsyms checkSat se
          else
            pure []
 
-      let execFeatures = psatf
+      simpleLoopFixpointFeature <- maybeToList <$> mapM
+        (\func -> liftIO $ Crucible.LLVM.Fixpoint.simpleLoopFixpoint sym cfg mvar $ \fixpoint_substitution condition ->
+          do let fixpoint_substitution_as_list = reverse $ MapF.toList fixpoint_substitution
+             let body_exprs = map (mapSome $ Crucible.LLVM.Fixpoint.bodyValue) (MapF.elems fixpoint_substitution)
+             let uninterpreted_constants = foldMap
+                   (viewSome $ Set.map (mapSome $ W4.varExpr sym) . W4.exprUninterpConstants sym)
+                   (Some condition : body_exprs)
+             let filtered_uninterpreted_constants = Set.toList $ Set.filter
```

Comment:
@andreistefanescu, can you say a little bit about what's going on here with this filtering function?